.github/workflows/teste2e.yml

```diff
@@ -33,3 +33,13 @@ jobs:
         env:
           TEST_CLIENT_DOMAIN: ${{secrets.TEST_CLIENT_DOMAIN}}
           TEST_ID: ${{steps.random-path-generator.outputs.TEST_ID}}
+      - name: Report Status
+        if: always()
+        uses: ravsamhq/notify-slack-action@v1
+        with:
+          token: ${{ secrets.GH_RELEASE_TOKEN }}
```

Comment:
Do we really need to pass our token to this action?

Reply:
Probably not, I'll try now without our token